package.json

```diff
@@ -103,7 +107,10 @@
     "parser": "babel-eslint",
     "extends": "airbnb",
     "rules": {
-      "max-len": ["error", 130]
+      "max-len": [
+        "error",
+        130
+      ]
```

Comment:
It's corrected by my text editor automatically according to rule [4.6](https://github.com/airbnb/javascript#arrays--bracket-newline) of Airbnb JavaScript Style Guide.